### PR TITLE
PMM-5728 Management command for external services.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: commands/management/add_external.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+			Username:       cmd.Username,
+			Password:       cmd.Password,
+			Scheme:         cmd.Scheme,
+			MetricPath:     cmd.MetricPath,
+			ListenPort:     int64(cmd.ListenPort),
+			NodeID:         cmd.NodeID,
+			Environment:    cmd.Environment,
+			Cluster:        cmd.Cluster,
+			ReplicationSet: cmd.ReplicationSet,
+			CustomLabels:   customLabels,
```
Comment (askomorokhov) [inline commands/management/add_external.go:92]: Shouldn't be this a list of strings?

File: commands/management/add_external.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+var (
+	AddExternal  = new(addExternalCommand)
+	AddExternalC = AddC.Command("external", "Add External to monitoring")
+)
+
+func init() {
+	hostname, _ := os.Hostname()
+	serviceName := hostname + "-external"
+	serviceNameHelp := fmt.Sprintf("Service name (autodetected default: %s)", serviceName)
+	AddExternalC.Flag("service-name", serviceNameHelp).Default(serviceName).StringVar(&AddExternal.ServiceName)
```
Comment (askomorokhov) [inline commands/management/add_external.go:116]: Looks like this flag conflicts with the same in add.go

**Review by askomorokhov — CHANGES_REQUESTED** [review]

File: commands/management/add_external.go (hunk 1 above)
Comment (BupycHuk) [reply commands/management/add_external.go:92]: no, it's a map and we have common logic for all commands to parse custom labels in pmm-admin.

File: commands/management/add_external.go (hunk 2 above)
Comment (BupycHuk) [reply commands/management/add_external.go:116]: good point

**Review by askomorokhov — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

**Pull request merged**